packages/rancher-istio-1.8/scripts/version_update.go

```diff
@@ -1,60 +0,0 @@
-package main
```

Comment:
To hard to maintain this ?

Reply:
Yes. Also there are fewer versions to change when doing an upgrade (4 images and 2 spots in the chart.yaml) so the script seemed overkill when comparing effort to change version vs effort to maintain the script.